### PR TITLE
BVAL-695 Addressing misc. findings from review

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://hibernate.atlassian.net/browse/BVAL-695

### CODE REVIEW

File: spec-examples/src/test/java/org/beanvalidation/specexamples/constraintmetadata/EmployeeImpl.java [hunk 1]
```diff
@@ -32,7 +29,7 @@ public Roles getRoles() {
 	}
 
 	@Override
-	public Map<@NotNull String, @Valid Account> getAccounts() {
```
Comment (gunnarmorling) [inline spec-examples/src/test/java/org/beanvalidation/specexamples/constraintmetadata/EmployeeImpl.java]: Would be nice to double-check on this - they seemed redundant to me as they are already given on the overridden super-type method.

File: sources/value-extractor-definition.asciidoc [hunk 2]
```diff
@@ -160,8 +160,8 @@ public class OptionalValueExtractor implements ValueExtractor<Optional<@Extracte
 
 This extractor passes the element wrapped by the given `Optional` to the receiver object, if present.
 `null` is passed as a node name, causing no node to be appended to the resulting property path.
-I.e. when the `@Size` constraint in `private Optional<@Size(min=1) String name;` is violated,
-the resulting property path will be the same as if a constraint hosted on the `name` field itself was violated.
+I.e. when the `@Size` constraint in `Optional<@Size(min=1) String getName() { ... }` is violated,
```
Comment (gsmet) [inline sources/value-extractor-definition.asciidoc]: Mmmh `getName` property feels weird. Wouldn't it be the `name` property anyway?

File: spec-examples/src/test/java/org/beanvalidation/specexamples/constraintmetadata/EmployeeImpl.java (hunk 1 above)
Comment (gsmet) [inline spec-examples/src/test/java/org/beanvalidation/specexamples/constraintmetadata/EmployeeImpl.java]: Looks good.

**Comment by gsmet** [comment]
Amended and merged, thanks!

**Pull request closed without merging**